utils/caching/purgeMiddleware.ts

```diff
@@ -126,29 +110,11 @@ export const purgeMiddleware = (
 			}
 
 			defer(async () => {
-				if (req.path === '/api/users' && req.method === 'PUT') {
-					const hostnames = await getUniqueHostnamesForUserId(req.user.id);
-
-					const allPurges = [
-						// all the communities the user is a member of,
-						// either directly or through a pub/collection
-						...hostnames,
-						// the current community
-						surrogateTag,
-						// purge the cache for the logged in user
-						getPPLic(req.user),
-						// purge all the /user pages
-						req.user.id,
-					];
-
-					await Promise.all(allPurges.map(async (tag) => schedulePurge(tag)));
-				} else {
-					/**
-					 * this await is here so you can wait on the purge requests
-					 * using `finishDeferredTasks` in tests or other short lived processes
-					 */
-					await schedulePurge(surrogateTag);
-				}
```

Comment:
this was moved here: https://github.com/pubpub/pubpub/pull/2963/files#diff-091fe153990dde352868e0d97e0be2e9ebff522544644952f1348961f6cceaab